randovania/data/json_data/prime3.txt

```diff
@@ -3684,15 +4527,18 @@ Leads to Command Center/Dock to Metroid Creche
       Trivial
 
 > Event - Metroid Hatcher; Heals? False
-  > Pickup (Energy Tank)
-      Spider Ball and Boost Ball
+  * Event Metroid Hatcher (Pirate Homeworld)
+  > Bottom Floor
+      Trivial
 
 > Bottom Floor; Heals? False
   > Dock to Creche Transit
       Trivial
   > Dock to Transit Station 3-A
 Leads to Command Center
       Use Screw Attack (Space Jump)
+  > Pickup (Energy Tank)
+      Spider Ball and Boost Ball and After Metroid Hatcher (Pirate Homeworld)
   > Event - Metroid Hatcher
       All of the following:
           Boost Ball
```

Comment:
For Scrapworks (Below):

- Remove the two helper nodes
- Change all dock nodes to Morph Ball Doors
- First Dock
  - Rename to "Morph Ball Door to Scrapvault (X-Ray Terminal)"
  - Connect to "Morph Ball Door to Scrapvault (Exit)" with Boost
- Second Dock
  - Rename to "Morph Ball Door to Scrapvault (Entry)"
  - Connect to Etank as trivial
  - Connect to "Morph Ball Door to Scrapvault (X-Ray Terminal)" as trivial
  - Connect to "Morph Ball Door to Scrapvault (Exit)" with Spider
- Third Dock
  - Rename to "Morph Ball Door to Scrapvault (Exit)"
  - Connect to Ship Missile with Boost and Spider
  - Connect to "Morph Ball Door to Scrapvault (X-Ray Terminal)" with Boost
  - Connect to "Morph Ball Door to Scrapvault (Entry)" with Boost
- Etank
  - Connect to "Morph Ball Door to Scrapvault (Entry)" as trivial
- Ship Missile
  - Connect to "Morph Ball Door to Scrapvault (Exit)" as trivial